lib/telemetry_metrics_cloudwatch.ex

```diff
@@ -145,12 +159,19 @@ defmodule TelemetryMetricsCloudwatch do
   end
 
   @impl true
-  def handle_info({:handle_event, measurements, metadata, metrics}, state) do
+  def handle_info(
+        {:handle_event, measurements, metadata, metrics},
+        %Cache{sample_rate: sample_rate} = state
+      ) do
```

Comment:
Not a big deal, but I think the formatting looks a little nicer this way since there's no parameter matching in the function definition:

```suggestion
  def handle_info({:handle_event, measurements, metadata, metrics}, state) do
      %Cache{sample_rate: sample_rate} = state
```